# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnomemm12/gcc4.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gnomemm-1.2.4/src/gnome--/app-helper.h.orig	2003-01-27 22:15:30.000000000 +0100
+++ gnomemm-1.2.4/src/gnome--/app-helper.h	2005-09-23 12:25:32.018185797 +0200
@@ -40,6 +40,7 @@
 template<class T_Info>
 class Array;
 
+class Info_;
 class Info : protected GnomeUIInfo
 {
   /* Note when deriving this, you must not add any fields nor may you
